components/application_manager/application_manager.cpp

```diff
@@ -924,13 +866,7 @@ void ApplicationManager::OnPerformBroadcastAutostart()
     if (app_desc != nullptr)
     {
         LOG(LOG_ERROR, "OnPerformAutostart Start autostart app.");
-
-        auto newApp = App::CreateAppFromAitDesc(app_desc, m_currentService, m_isNetworkAvailable,
-            "", true, false);
-        if (!RunApp(newApp))
-        {
-            LOG(LOG_ERROR, "OnPerformAutostart Failed to create autostart app.");
-        }
+        CreateAndRunApp(*app_desc, "", true, false);
```

Comment:
NITPICK: Should this do something with the returned boolean? Although it does some logging internally. Not sure.

Reply:
From Marios: "TBH, I don't really care whether the returned value gets discarded. I don't think it is necessary to do something with it always."